requirements.txt

```diff
@@ -233,6 +233,7 @@ nest-asyncio==1.5.6
     #   jupyter-client
 nodeenv==1.7.0
     # via pre-commit
+nox
```

Comment:
Remember that you should pin a version here! The command to do that is up top, but it won't work until you add nox to `pyproject.toml` first.